Intersect.Server.Core/Entities/Player.cs

```diff
@@ -5438,36 +5437,16 @@ public void UseSpell(int spellSlot, Entity target)
                 // retargeting for auto self-cast on friendly when targeting hostile
                 CastTarget = target;
 
-                //Check if the caster has the right ammunition if a projectile
-                if (spell.SpellType == SpellType.CombatSpell &&
-                    spell.Combat.TargetType == SpellTargetType.Projectile &&
-                    spell.Combat.ProjectileId != Guid.Empty)
-                {
-                    var projectileBase = spell.Combat.Projectile;
-                    if (projectileBase != null && projectileBase.AmmoItemId != Guid.Empty)
-                    {
-                        TryTakeItem(projectileBase.AmmoItemId, projectileBase.AmmoRequired);
-                    }
-                }
-
```

Comment:
Moved this out to the override void CastSpell because this was removing projectiles before we could even cast them out properly, which leads to the actual bug of not being able to properly use the last projectile